tests/robot/_resources/test_data_sets/query/expected_results/loaded_db/B/803_get_composition_by_uid.json

```diff
@@ -2,9 +2,9 @@
   "meta": {
     "_type": "RESULTSET",
     "_created": "2019-10-18T18:12:29.282Z",
-    "_executed_aql": "SELECT c FROM COMPOSITION c WHERE c/uid/value=__MODIFY_COMPOSITION_UID_1__"
+    "_executed_aql": "SELECT c FROM COMPOSITION c WHERE c/uid/value='__MODIFY_COMPOSITION_UID_1__'"
   },
-  "q": "SELECT c FROM COMPOSITION c WHERE c/uid/value=$uid",
+  "q": "SELECT c FROM COMPOSITION c WHERE c/uid/value='__MODIFY_COMPOSITION_UID_1__'",
```

Comment:
Here, $uid should be in the query "q" since it's the AQL before the substitution of the parameter value.

Reply:
I reverted my mistaken changes.